offices/_single.html

```diff
@@ -147,11 +143,7 @@ <h1 class="h2">
                 <div class="expandable_content">
                     {% if page.short_title or page.title %}
                     <h2 class="h4 u-hide-on-mobile">
-                        {% if page.short_title %}
-                            {{ page.short_title | safe }}
-                        {% elif page.title %}
-                            {{ page.title | safe }}
-                        {% endif %}
+                        {{ page.short_title|safe if page.short_title else page.title|safe }}
```

Comment:
On this and on line 131 I changed this logic. It does the same thing. If anyone wants the older version, just say so and I'll change it.